src/app/shared/utils/http-util.ts

```diff
@@ -37,4 +39,39 @@ export class HttpUtil {
       })
     );
   }
+
+  static getRequestObservables(
+    httpRequestObservable: Observable<any>
+  ): {
+    response$: Observable<any>;
+    loading$: Observable<boolean>;
+    error$: Observable<string>;
+  } {
+    const data$ = httpRequestObservable.pipe(shareReplay(1));
+
+    const response$ = data$.pipe(
+      filter((res) => !res._status),
```

Comment:
where does this `_status` come from? do correct responses not have a `_status` property?